scipy/stats/_binned_statistic.py

```diff
@@ -459,6 +459,44 @@ def binned_statistic_dd(sample, values, statistic='mean',
 
     .. versionadded:: 0.11.0
 
+    Examples
+    --------
+    >>> import numpy as np
```

Comment:
If I'm not mistaken, I think our documentation testing machinery will automatically provide `np`, so we don't need this.